src/Plugin/EntityReferenceSelection/OgSelection.php

```diff
@@ -25,7 +25,7 @@
  * @EntityReferenceSelection(
  *   id = "og:default",
  *   label = @Translation("OG selection"),
- *   group = "default",
+ *   group = "og",
```

Comment:
Yes, but if we don't have the field, we can't get a selection handler based on field. For me this is happening when trying to create any node or user. Probably any other entity possibly that will use the valid reference constraint.